assets/RNAseq_report.Rmd

```diff
@@ -2160,20 +2209,17 @@ if (isProvided(params$custom_gmt)){
     for (db in datasources) {
         database_string <- paste0(database_string, ", `", db, ": ", database_version_info$sources[[db]]$version, "`")
     }
-    database_string <- paste0(database_string, ".") # Add final dot to sentence
 }
 
 cat(paste0(
-"For pathway analysis, the R packages `gprofiler2 v",
-version_gprofiler2,
-" `, `AnnotationDbi v",
-version_annotation,
-"` and `",
-name_species,
-" v",
-version_annotation,
-"` were employed. ",
-database_string))
+"\n## Pathway enrichment analysis
+
+For pathway analysis, the R packages `gprofiler2 v", version_gprofiler2,
+" `, `AnnotationDbi v", version_annotation,
+"` and `", name_species, " v", version_annotation,
+"` were employed. ", database_string, ".\n",
+"Pathways were classified as enriched for those genes with padj <= ", pval_text, "."
```

Comment:
```suggestion
"Pathways were classified as enriched for genes with padj <= ", pval_text, "."
```

Reply:
counter proposal: `an adjusted p-value <= pval_test`? I prefer having <= (or alternatively "less than or equal to") in the text so people don't wonder, even if that is an edge case